packages/next/src/server/async-storage/cache-scope.external.ts

```diff
@@ -1,11 +1,10 @@
-import { createAsyncLocalStorage } from '../app-render/async-local-storage'
+import { cacheScopeAsyncLocalStorage } from './cache-scope-instance' with { 'turbopack-transition': 'next-shared' }
 
 export interface CacheScopeStore {
-  cache?: Map<string, any>
+  cache: Map<string, any>
```

Comment:
Since this is untyped and the keys don't conflict with fetch cache I use the same Map but we should probably split this into two separate Maps that have different types of entries.